Userland/Utilities/mediaprobe.cpp

```diff
@@ -0,0 +1,150 @@
+
+
+#include <AK/String.h>
+#include <LibCore/ArgsParser.h>
+#include <LibMedia/Manip.h>
+#include <LibMedia/readers/AVI/AVI.h>
+#include <iomanip>
+#include <iostream>
+
+using Media::u32_to_fourcc;
+
+int main(const int argc, char** argv)
+{
+    String filePath;
+    bool index { false };
+    bool verbose { false };
+    bool veryVerbose { false };
+
+    Core::ArgsParser args_parser;
+    args_parser.set_general_help("Display information about an audio/video file");
+    args_parser.add_option(index, "Index tracks", "index", 'i');
+    args_parser.add_option(verbose, "Verbose output", "verbose", 'v');
+    args_parser.add_option(veryVerbose, "Very verbose output", "VeryVerbose", 'V');
+    args_parser.add_positional_argument(filePath, "Path to file", "path");
+    args_parser.parse(argc, argv);
+
+    // FIXME: get the reader type dynamically instead of hardcoding.
+    Media::Reader::AVI::AVIReader reader(filePath);
+    if (reader.is_open() == false) {
+        std::cerr << "Failed to open file" << std::endl;
+        return 2;
+    }
+
+    if (veryVerbose == true) {
+        verbose = true;
+    }
+
+    std::cout << "General" << std::endl;
+    std::cout << "Complete name\t\t: " << filePath.characters() << std::endl;
```

Comment:
First comment, don't use STL headers or constructs at all please.

`<iomanip>` and `<iostream>` should be replaced with `AK/Format.h` , `std::cout` should be replaced with `outln`, and `std::cerr` should be replaced with `warnln`. The formatting should be replaced with the AK::Format (fmtlib/std::format) style of formatting strings.

```
outln("General");
outln("Complete name\t\t: {}", file_path);
```
etc.

Also, per the [Coding Style Guide]( https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md), all locals and such should be snake_case. No braces on single-line if statements, etc. Please review it and run clang-format-11 or clang-format-11 over this code :)

Reply:
Thanks. I had been advised in Discord that wasn't the case and to use stl headers as a last resort.

I did have a FIXME elsewhere about this because I couldn't identify an equivalent for iostream/iomanip. I know there is at least 1 other instance elsewhere where an stl header is used. Is there an equivalent for std::to_string(int) or would AK/Format.h cover that?

Reply:
`String::number(...)`

Reply:
> From the [contributing guidelines](https://github.com/SerenityOS/serenity/blob/cd52eb3103456f36bf09d07bf1a3d3ed1fb5c5f0/CONTRIBUTING.md):
> 
> > For your first couple of PR's, start with something small to get familiar with the project and its development processes. Please do not start by adding a new application, library or other large component.

I appreciate this point but I have done so in the past, just not on this account.

Reply:
@ADKaster I have ran clang-format-11 using the .clang-format file over it all. Is there something it has missed?

Reply:
@IdanHo cheers. Turns out I didn't need it in the end.

Reply:
I think my eyes were just tricked by the rendering of the tabs in some of your files in the GitHub UI. So, replacing them with 4 spaces like the rest of the project would probably make it clearer that the style is correct :)

Reply:
Hmm, that's odd. I turned on whitespace in my editor and only 1 file has tabs in. Is it showing tabs for all in Github UI for you?